config.py

```
@@ -77,9 +242,33 @@ class DeviceMeta(TypedDict):
+    "12": "oneplus_12",
+    "12R": "oneplus_12r",
+    "11": "oneplus_11",
+    "11R": "oneplus_11r",
+    "10 Pro": "oneplus_10_pro",
+    "10T": "oneplus_10t",
+    "10R": "oneplus_10r_80w",
+    "9 Pro": "oneplus_9_pro",
+    "9": "oneplus_9",
+    "9RT": "oneplus_9rt",
+    "9R": "oneplus_9r",
+    "9R": "oneplus_9r",
```

Ruff (0.14.14)

[error] Dictionary key literal `"9R"` repeated

Remove repeated key literal `"9R"`

(F601)